suave/sol/libraries/Suave.sol

```diff
@@ -218,8 +219,8 @@ library Suave {
         return abi.decode(data, (DataRecord));
     }
 
-    function privateKeyGen() internal view returns (string memory) {
-        (bool success, bytes memory data) = PRIVATE_KEY_GEN.staticcall(abi.encode());
+    function privateKeyGen() internal returns (string memory) {
+        (bool success, bytes memory data) = PRIVATE_KEY_GEN.call(abi.encode());
```

Comment:
i believe this is on purpose?

Reply:
I didn't change it; this is the output of Ferran's codegen tool.  I do believe there was a recent PR to make this a dynamic call, though.

Reply:
Resolving and merging, since you approved :)